docs/current/README.md

```diff
@@ -17,37 +17,125 @@
 </head><article class="markdown-body">
 
 # Vaquero
+[Home](https://ciscocloud.github.io/vaquero-docs/)
+
 [![Build Status](https://drone.projectshipped.io/api/badges/CiscoCloud/vaquero/status.svg)](https://drone.projectshipped.io/CiscoCloud/vaquero)
 
-A bare metal configuration tool that leverages github templates, and CoreOS bare-metal to net boot machines.
+- [Repo](https://github.com/CiscoCloud/vaquero) : Where development of vaquero is taking place.
+- [Waffle.io Issue Tracking](https://waffle.io/CiscoCloud/vaquero): How the team is tracking work and progress
+
+A bare metal configuration tool that takes in templates that describe a data center and automatically network boot those machines into the desired state.
 
 # High Level Overview
 
 ## [Architecture](https://ciscocloud.github.io/vaquero-docs/docs/current/architecture.html)
-![](https://raw.githubusercontent.com/CiscoCloud/vaquero-docs/gh-pages/docs/current/architecturediagram.png)
+![](https://raw.githubusercontent.com/CiscoCloud/vaquero-docs/docs-cleanup/docs/current/ppt-arch.png)
 
-## [Data Model Templates](https://github.com/CiscoCloud/vaquero-docs/blob/gh-pages/docs/current/env-data-structure.md)
+## [Data Model Templates](https://ciscocloud.github.io/vaquero-docs/docs/current/data-model-howto.html)
+- [Example Data Models](https://github.com/gem-test/vaquero)
```

Comment:
I think we should shoot for master to be a good reference. I'll also reference local as its the one I use for functional tests and vagrant for vagrant.